tests/torch-ort_test_api.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import _test_helpers
+
+import copy
+import onnxruntime as ort
+import torch
+
+from onnxruntime.training import ORTModule
+
+
+class NeuralNetSinglePositionalArgument(torch.nn.Module):
+    def __init__(self, input_size, hidden_size, num_classes):
+        super(NeuralNetSinglePositionalArgument, self).__init__()
+
+        self.fc1 = torch.nn.Linear(input_size, hidden_size)
+        self.relu = torch.nn.ReLU()
+        self.fc2 = torch.nn.Linear(hidden_size, num_classes)
+
+    def forward(self, input1):
```

Comment:
There is no randomness in this model, let's add a dropout layer.

Reply:
Added a dropout layer